index.js

```diff
@@ -19,9 +19,12 @@ const exampleProducts = [
   - The `cart` array is empty.
 */
 function getCartTotal(cart) {
+  if(!cart.length){//guard clause
+    throw "";
```

Comment:
```suggestion
    throw "Cart is empty.";
```
 It's good to have descriptive messages for each type of error, but since this is just a practice lab, it's okay that you just have "". Here's an example of a descriptive error message that you could use instead of an empty string.